shared/fetcher.js

```diff
@@ -235,7 +235,9 @@ Fetcher.prototype.bootstrapData = function(modelMap) {
       cb(null);
     });
   }, function(err) {
-    fetcher.storeResults(results);
```

Comment:
We don't need to manually call the `store` method because it get called each time a new model or collection is created.  (https://github.com/rendrjs/rendr/blob/master/shared/base/model.js#L25)